### PR TITLE
Fix licensing

### PR DESCRIPTION
**Pull request by Meziu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# shim-3ds
```
Comment (ian-h-chamberlain) [inline README.md:1]: I guess we can do a separate PR for renaming the crate based on #17

**Review by ian-h-chamberlain — APPROVED** [review]

**Review by Techie-Pi — APPROVED** [review]
